software-modules/programming/atom/atom.sh

```diff
@@ -18,12 +18,16 @@ set -xe
 NAME=atom
 TARGET_DIR="/run/initramfs/memory/system/huronOS/software/programming/"
 
+## Add atom keyring
+wget -qO - https://packagecloud.io/AtomEditor/atom/gpgkey | gpg --dearmor -o /usr/share/keyrings/atomeditor-keyring.gpg
+
+## Add repository to deb source list
+echo "deb [arch=amd64 signed-by=/usr/share/keyrings/atomeditor-keyring.gpg] https://packagecloud.io/AtomEditor/atom/any/ any main" | tee /etc/apt/sources.list.d/atom.list
```

Comment:
Is this repository trusted? I've never used it, but in my case I recovered Atom deb file from internet.org
Actually, probably we'll be needing to re-check if new IOI standards will still requiring this one because it's super deprecated editor.

Reply:
[It seems](https://webcache.googleusercontent.com/search?q=cache:9Xro-ZA9B9wJ:https://ioi2023.hu/contest-environment/&cd=9&hl=en&ct=clnk&gl=mx) like atom is still in use by the IOI